includes/class-suggestion-engine.php

```
@@ -0,0 +1,347 @@
+		 *
+		 * @since  1.6.0
+		 * @param  string $metric      Metric identifier.
+		 * @param  float  $score       Lighthouse score (0.0–1.0).
+		 * @param  string $fix_action  fix_action value.
+		 * @param  string $description Human-readable metric name.
+		 * @return array Suggestion object.
+		 */
+		private static function make_score( string $metric, float $score, string $fix_action, string $description ): array {
+			$status = $score >= 0.5 ? 'needs_improvement' : 'poor';
+			return self::build( $metric, $score, 'score', $status, $description, $fix_action );
+		}
```

PHPMD (2.15.0)

[error] The method make_score is not named in camelCase. (undefined)

(CamelCaseMethodName)